librz/analysis/class.c

```diff
@@ -478,13 +497,78 @@ static RzAnalysisClassErr rz_analysis_class_add_attr_unique(RzAnalysis *analysis
 // Format: addr,vtable_offset
 
 static char *flagname_method(const char *class_name, const char *meth_name) {
+	if (rz_str_startswith(meth_name, "method.")) {
+		return rz_str_new(meth_name);
```

Comment:
In which practical case is this branch taken?

Reply:
In `rz_analysis_class_method_recover()`, the method name is from `rz_analysis_get_function_at()` when this function was already analyzed and added to the context, which contains `method.`. But `flagname_method()` which is called in `rz_analysis_class_method_set()` will add the prefix `method.` again.

Reply:
Ok, looks good to me then.